### DISCUSSION TITLE
How to adjust workflow execution time?

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15318] [How to]
As per below I would like to change days to seconds. Where is the code for that?
The reason I asking is that multiple workflows on record creation are easily falling in conflict. 
Separating workflows On Create with various timing will allow  to run multiple workflows on record creation
Thanks :-)

![image](https://user-images.githubusercontent.com/84110093/137573979-5e88d039-f5b3-4bfa-99c9-272a8b57f87b.png)

**Comment by garryatbrighton** [comment]
would look for it

**Reply by garryatbrighton** [reply]
it could be done if workflows starts in designated order. Then list of workflows should be sortable. For instance all workflows on record creation can be sorted and run one after previous one completed. Would it be helpful or here is issues with idea?

**Comment by JohnMaverick** [comment]
this would be very useful. Currently I am experiencing problems with combination of workflows on recrord creating. They overlapping and blocking each other execution. Maybe developer can get some clue on how to prevent it?

**Comment by TonyM15** [comment]
Extending this to minutes or even seconds will be useful especially that there is no predefined workflows working with Date/Time field (only time field). For non programer setting "Time field" to Current time is impossible at the moment.
Alternatively adding predefined "Now"function working with "Time Field" would resolve this basic obstacle

**Comment by randysrs** [comment]
how do you set a time expression to a particular time of day . 
example 
want a email to go out on the 4th of july at 9am.

**Comment by Marshal4582** [comment]
I would follow up this thread. System indeed not allowing for one-time execution, nor more precise delays on record creation/modification.
This types of workflows ad very often used and are common in other simillar systems. 
Currently to crack on one time workflow we ned to create additional fields to stop Execution Schedule trigerring. The question hanging here: what if we have in module 30 mln records? Execution schedule could be a bottle neck, as it checking all records?

**Reply by TonyM15** [reply]
Hoping Yetiforce will consider that little improvement. It couple of clicks, but will do pretty good for the base strengthennig

**Comment by s2b-git** [comment]
I would like to bump this one. Can developer point out where to change days to seconds as per below? :-)

![image](https://user-images.githubusercontent.com/84110093/169641465-03236562-2c3c-481a-8877-7f48ed826581.png)

**Reply by mariuszkrzaczkowski** [reply]
create a separate discussion

**Reply by s2b-git** [reply]
#16078
